app/src/main/res/values/strings.xml

```diff
@@ -143,34 +144,34 @@
     <string name="detail_item_menu_cancel">Cancel download</string>
     <string name="detail_item_menu_save_file">Save file</string>
     <string name="detail_item_menu_copy_url">Copy URL</string>
-    <string name="detail_item_menu_copy_url_copied">Copied URL to clipboard</string>
+    <string name="detail_item_menu_copy_url_copied">URL copied to clipboard</string>
     <string name="detail_item_menu_copy_contents">Copy notification</string>
-    <string name="detail_item_menu_copy_contents_copied">Copied notification clipboard</string>
-    <string name="detail_item_saved_successfully">Saved as %1$s to Downloads folder</string>
-    <string name="detail_item_cannot_download">Cannot open or download attachment. Link expired and no local file found.</string>
+    <string name="detail_item_menu_copy_contents_copied">Notification copied to clipboard</string>
+    <string name="detail_item_saved_successfully">Saved as %1$s in the \"Downloads\" folder</string>
+    <string name="detail_item_cannot_download">Cannot open or download attachment. The link expired and no local file could be found.</string>
     <string name="detail_item_cannot_open">Cannot open attachment: %1$s</string>
-    <string name="detail_item_cannot_open_not_found">Cannot open attachment: File may have been deleted, or there is no app to open the file.</string>
-    <string name="detail_item_cannot_open_click_url">Cannot open click URL: %1$s</string>
```

Comment:
(Not sure what a "click URL" is.)

Reply:
https://ntfy.sh/docs/publish/#click-action

It's the URL you defined in the `Click` header that defines what URL is opened when you click the notification.